demos/projects/ST/b-l475e-iot01a/port/azure_iot_flash_platform.c

```diff
@@ -37,18 +43,44 @@ static AzureIoTResult_t prvBase64Decode( uint8_t * base64Encoded,
         return eAzureIoTErrorFailed;
     }
 
-    LogInfo( ( "Unencoded the base64 encoding\r\n" ) );
+    AZLogInfo( ( "Unencoded the base64 encoding\r\n" ) );
 
     return eAzureIoTSuccess;
 }
 
 AzureIoTResult_t AzureIoTPlatform_Init( AzureADUImage_t * const pxAduImage )
 {
-    (void) pxAduImage;
-
-    // TODO: Fill in initialization for ST flash writing capability.
-
-    LogInfo( ( "AzureIoTPlatform_Init()\r\n" ) );
+    pxAduImage->xUpdatePartition = (uint8_t *)(FLASH_BASE + FLASH_BANK_SIZE);
+    pxAduImage->pucBufferToWrite = NULL;
+    pxAduImage->ulBytesToWriteLength = 0;
+    pxAduImage->ulCurrentOffset = 0;
+    pxAduImage->ulImageFileSize = 0;
+
+    static FLASH_EraseInitTypeDef EraseInitStruct;
```

Comment:
```suggestion
    static FLASH_EraseInitTypeDef xEraseInitStruct;
```

Reply:
Is there a specific guideline we follow for naming conventions? After searching a little, I found [this](https://azure.github.io/azure-sdk/clang_design.html), but it seems to be a bit different for variables